google/cloud/aiplatform/training_jobs.py

```diff
@@ -1429,9 +1438,40 @@ def _prepare_training_task_inputs_and_output_dir(
             training_task_inputs["network"] = network
         if tensorboard:
             training_task_inputs["tensorboard"] = tensorboard
+        if enable_web_access:
+            training_task_inputs["enable_web_access"] = enable_web_access
 
         return training_task_inputs, base_output_dir
 
+    @property
+    def web_access_uris(self) -> Optional[Dict[str, str]]:
+        """Get the web access uris of the backing custom job.
+
+        Returns:
+            (Dict[str, str]):
+                Web access uris of the backing custom job.
+        """
+        if self._custom_job:
+            self._web_access_uris = self._custom_job.web_access_uris
```

Comment:
I don't think we need to assign `_web_access_uris` and maintain property since we are returning the value.